dp/generate_climos.py

```diff
@@ -169,11 +184,9 @@ def climo_outputs(time_resolution):
         cf.climo_start_time = t_start.isoformat()[:19] + 'Z'
         cf.climo_end_time = t_end.isoformat()[:19] + 'Z'
         if hasattr(input_file, 'tracking_id'):
-            cf.climo_tracking_id=input_file.tracking_id
-        try:
-            cf.variables['pr'].units = pr_units_attr
-        except NameError:
```

Comment:
Silent errors are definitely bad, so I approve of getting rid of this. What actually happens here if the variable is not attributed with units? I guess we're assuming that everything going through this script passes nclint and therefore *is* attributed with units, eh? I'm fine with that, just making sure.